static/riotjs/wizard/participants_table.tag

```diff
@@ -65,10 +65,18 @@
 
     <div style="">
         <span>
-            <a onclick="{download_csv}" class="ui yellow button">Download CSV</a>
-            <a class="ui blue button" data-tooltip="Format: First Name,Last Name,Display Name,Student ID,Email,Team,TeamLeader(Optional bool)"
+            <div onclick="{download_csv}"
+                 data-tooltip="Download CSV of the students and their team assignments in this class."
+                 class="ui blue button">Download Students CSV</div>
+            <a href="/api/v1/enroll_students_sample_csv/?format=csv"
+               class="ui blue button"
+               download="sample_student_enroll.csv"
+               data-tooltip="Download this CSV as an example template for the format required to upload a CSV of students.
+                             This CSV is populated by dummy data.">
+                Download Template Students CSV
+            </a>
+            <a class="ui facebook button" data-tooltip="Column Format: First Name,Last Name,Display Name,Student ID,Email,Team,Team Leader(Optional boolean)"
```

Comment:
Why do we use the facebook button class here out of curiosity?

Reply:
Just for another shade of blue. Trying to slightly differentiate this from standard `blue` to show that it's related.